### ISSUE TITLE
dwds failing tests on windows

### ISSUE THREAD

**Issue by jakemac53** [dart-lang/webdev#711]
Some example failures below:

test\dart_uri_file_uri_test.dart: file path to org-dartlang-app

 ```
Invalid argument (uri): Unknown library: "file:///C:/..."
```

All these breakpoint tests fail with what looks like the same issue:

test\chrome_proxy_service_test.dart: shared context breakpoints addBreakpoint
test\chrome_proxy_service_test.dart: shared context breakpoints addBreakpoint on a part file
test\chrome_proxy_service_test.dart: shared context breakpoints addBreakpointWithScriptUri

```
NoSuchMethodError: The getter 'id' was called on null.
  Receiver: null
  Tried calling: id
  dart:core                                  Object.noSuchMethod
  test\chrome_proxy_service_test.dart 57:55  main.<fn>.<fn>.<fn>
```
And a lot more...

**Comment by alan-knight** [comment]
Fixed a number of these in #717

**Comment by grouma** [comment]
There are a couple skipped tests left but most windows tests have been fixed / added.
